client/src/containers/AuthContainer.tsx

```diff
@@ -49,63 +57,66 @@ function AuthContainer() {
 
   return (
     <div className="AuthContainer">
-      <Grid
-        container
-        component="main"
-        justifyContent="flex-end"
-        alignItems="center"
-        sx={{ height: "100vh", px: "40px" }}
-      >
-        <Grid item hidden={matches} xs={false} md={8}>
-          <Box
-            sx={{
-              display: "flex",
-              flexDirection: "column",
-              textAlign: "center",
-              alignItems: "center",
-            }}
-          >
-            <Box bgcolor="rgba(83, 102, 133, 0.6)" p="40px" px="80px">
-              <Typography color="white" fontFamily="Verdana" variant="h2">
-                UBoard
-              </Typography>
-              <Typography color="white" fontFamily="Verdana" variant="h4">
-                A place to connect
-              </Typography>
-            </Box>
-            
-          </Box>
-        </Grid>
+      <ThemeProvider theme={theme}>
```

Comment:
The theme provider should be at the root as the theme is applied application wide. So it should wrap all routes - not just this one.

Reply:
Ok, I think should Pouya might have to update his PR as well then, since it looks like he might be overriding the main theme here.

Reply:
Ok